App/HomePage.js

```diff
@@ -184,12 +184,14 @@ class Homepage extends React.Component {
   }
 
   goToVenueRequests() {
-    console.log(this.props);
-    console.log('props of homepage ' + this.props.selectedIndex);
+
+    console.log('props of homepage ' + this.props);
```

Comment:
Brent: 
A few thoughts
You're binding this.goToVenueRequests twice in the constructor, could be a source of issues.
That said, you're overriding the bind of Homepage's scope (bind(this)) in this line: